kong/plugins/correlation-id/handler.lua

```diff
@@ -10,14 +10,26 @@ local CorrelationIdHandler = BasePlugin:extend()
 local worker_uuid
 local worker_counter
 
+local fmt = string.format
+
 local generators = setmetatable({
-	["uuid"] = function()
+  ["uuid"] = function()
     return uuid()
-	end,
-	["uuid#counter"] = function()
+  end,
+  ["uuid#counter"] = function()
     worker_counter = worker_counter + 1
     return worker_uuid.."#"..worker_counter
-	end,
+  end,
+  ["tracker"] = function()
+    return fmt("%s-%s-%s-%s-%s-%s",
+      ngx.var.server_addr,
+      ngx.var.server_port,
+      ngx.worker.pid(),
+      ngx.var.connection, -- connection serial number
+      ngx.var.connection_requests, -- current number of requests made through a connection
+      ngx.now() -- the current time stamp from the nginx cached time.
```

Comment:
I see you cached `string.format`, but please do follow my other caching suggestions.